.pairs

```diff
@@ -7,6 +7,7 @@ pairs:
   ln: Lasse Norfeldt; ln
   gh: Geoff Hubbard; gh
   tbb: Thor-Bjørn Böhme; tb
+  pb: Pierre-Arnaud Baciocchini; pb
```

Comment:
Unrelated? `.pairs` edit

Reply:
Agreed. Otherwise good.

Reply:
Fine with this. Also happy to remove and use github's co-authoring https://help.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors